.pre-commit-config.yaml

```diff
@@ -177,6 +177,11 @@ repos:
       - id: fix-encoding-pragma
         args:
           - --remove
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.7.2
```

Comment:
LGTM.

A minor question: shall we use the latest v2.7.3?

Reply:
Good call, will update